packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -109,6 +109,8 @@ function _SignUpStart(): JSX.Element {
     signUp
       .create({ strategy: 'ticket', ticket: formState.ticket.value })
       .then(signUp => {
+        removeClerkQueryParam('__clerk_ticket');
+        removeClerkQueryParam('__clerk_invitation_token');
```

Comment:
Is this the only place where the ticket can be consumed? 
Don't we need this to be moved in the resources themselves? Think about users using clerk-js directly :)

Reply:
Also, don't forget the unit tests

Reply:
The resource accepts the ticket as a param, it does not read it through the URL.

Reply:
Cool - let's add a unit test and merge